src/dialog/llm/__init__.py

```diff
@@ -20,5 +26,40 @@ def get_llm_class() -> Type[AbstractLLM]:
         llm_class = getattr(llm_module, class_name)
     except Exception as e:
         logging.info(f"Failed to load LLM class {Settings().LLM_CLASS}. Using default LLM. Exception: {e}")
+        raise
+
+    if isinstance(llm_class, AbstractLLM):
+        return llm_class, "AbstractLLM"
+
+    elif 'langchain_core.runnables' in str(type(llm_class)):
+        return llm_class, "LCELRunnable"
```

Comment:
why not check instance types directly instead of returning one more variable just for string comparison? For example, runnables can be checked like (using the `RunnableWithMessageHistory` defined in your `lcel.py` implementation)
```
In [76]: issubclass(RunnableWithMessageHistory, Runnable)
Out[76]: True
```

Reply:
I need to check on the computer why I've done this way.